packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -56,7 +57,7 @@ export const FileUploadButton = (props: FileUploadButtonProps): JSX.Element => {
       background: 'transparent'
     }
   });
-
+  const uploadFileString = useLocale().strings.chat.uploadFileButton;
```

Comment:
Naming the variable `uploadFileString` and referencing `uploadFileButton` is very confusing.
Let's use consistent names.

Reply:
done